src/overlays/actors/ovl_En_Hs/z_en_hs.h

```diff
@@ -8,9 +8,10 @@ struct EnHs;
 typedef void (*EnHsActionFunc)(struct EnHs*, GlobalContext*);
 
 typedef struct EnHs {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x250];
-    /* 0x0394 */ EnHsActionFunc actionFunc;
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ char unk_144[0x160];
+    /* 0x2A4 */ Vec3f nwcPos[20]; // positions of chicks during breman march
```

Comment:
Yeah this is a weird case.

The chick count is stored in a home z rotation, and is gets incremented by 2 rather than 1
/homhttps://github.com/zeldaret/mm/blob/8ff9e439f0f0a85803ab453f4a9969dcf8124fe6/src/overlays/actors/ovl_En_Nwc/z_en_nwc.c#L324
And then the array gets accessed with -2, so it climbs to 20 where the positions are the odd values
https://github.com/zeldaret/mm/blob/8ff9e439f0f0a85803ab453f4a9969dcf8124fe6/src/overlays/actors/ovl_En_Nwc/z_en_nwc.c#L324
I don't know why we are incrementing by 2, It's possible that the array is really an array of structs, where the other half of the array is some other value, will need to wait for grog decomp to know for sure.

Reply:
Aaaaand grog's code doesn't help us, it reads the odd indexes of the array but doesn't write them? So they are... undefined data? Guess I need to load in-game and see what Grog sees in that spot.